tests/test_burst.py

```diff
@@ -173,18 +173,43 @@ def test_get_burst_params_multiple_results():
 
 
 def test_validate_bursts():
```

Comment:
I don't see a test case for the `Reference granules must be from one date and secondary granules must be from one date` error, did I miss it?

Reply:
Add one!